apps/projects/app/components/Content/FilterTile.js

```diff
@@ -11,30 +12,38 @@ export default class FilterTile extends PureComponent {
   }
   render() {
     return (
-      <div style={{ margin: '1px 4px 1px 0' }}>
+      <Tile>
         <Badge.Identity>
-          <div style={{
-            paddingLeft: '10px',
-            paddingTop: '4px',
-            paddingBottom: '4px',
-            display: 'flex',
-            flexDirection: 'row'
-          }}>
+          <div
+            style={{
+              paddingLeft: '10px',
+              paddingTop: '4px',
+              paddingBottom: '4px',
+              display: 'flex',
+              flexDirection: 'row',
+            }}
+          >
             {this.props.text}
             <div
               style={{
                 marginLeft: '10px',
                 paddingRight: '4px',
-                cursor: 'pointer'
+                cursor: 'pointer',
               }}
-              role="button" aria-pressed="false"
+              role="button"
+              aria-pressed="false"
               onClick={this.props.disableFilter}
             >
-              <IconX height='8px' width='8px' />
+              <IconX height="8px" width="8px" />
             </div>
           </div>
         </Badge.Identity>
-      </div>
+      </Tile>
     )
   }
 }
+
+const Tile = styled.div`
+  display: inline-block;
```

Comment:
This is the reason I had to had to edit this file—to add `display: inline-block` to the tile. This allows these tiles to all stack in side-by-side like this:

<img width="1182" alt="projects with many filters added" src="https://user-images.githubusercontent.com/221614/59108575-39445000-8909-11e9-9640-674878cb6902.png">